modules/core/02-client/legacy/v100/solomachine.go

```diff
@@ -88,6 +88,13 @@ func (cs ClientState) ExportMetadata(_ sdk.KVStore) []exported.GenesisMetadata {
 	panic("legacy solo machine is deprecated!")
 }
 
+// VerifyClientMessage panics
```

Comment:
Mega nit, add the `!` to godoc for consistency

```suggestion
// VerifyClientMessage panics!
```